app/assets/stylesheets/base.scss

```diff
@@ -74,3 +74,30 @@ pre { word-wrap: break-word; }
 .badge-important {
   background-color: $red;
 }
+
+// grey card footer
+.card_footer {
+  position: absolute;
```

Comment:
Properties should be ordered background-color, border-top, bottom, font-size, left, line-height, margin-left, min-height, position, width